_posts/2017-10-25-geopandas-tutorial.md

```diff
@@ -221,20 +221,22 @@ aus_nz = {  "type": "Polygon",
             ]}
 {% endhighlight %}
 
-With this result, we should be able to intersect that cleaned continents layer and keep only the areas that we want that are relevant to the countries of interest. With this subset, we can perform the recursive Transit.Land search to get all Transit.Land operators that lie within those bounds as well.
+With this result, we should be able to intersect that cleaned continents layer and keep only the areas that we want that are relevant to the countries of interest. With this subset, we can perform the recursive Transitland search to get all Transitland operators that lie within those bounds as well.
 
-![aus_nz](https://raw.githubusercontent.com/kuanb/kuanb.github.io/master/images/_posts/transitland-boxes/aus_nz.png)
+![aus_nz](/images/geopandas-tutorial/aus_nz.png)
 
 Merging the two should result in the above image.
 
-![eurasia](https://raw.githubusercontent.com/kuanb/kuanb.github.io/master/images/_posts/transitland-boxes/eurasia.png)
+![eurasia](/images/geopandas-tutorial/eurasia.png)
 
 Above, another plot, this time of Europe and western Russia. As you can see, this pattern is easily reproducible anywhere.
 
 # Conclusion
 
-At this point you should be free to explore the feeds available on Transit.Land - there's a ton of them! Need ideas for what to explore next? How about choosing a more refined base layer if you'd like to work at a "closer" scale and explore and work with transit bounds data at, for example, the county and census tract scale in the United States.
+At this point you should be free to explore the feeds available on Transitland &mdash; there's a ton of them! Need ideas for what to explore next? How about choosing a more refined base layer if you'd like to work at a "closer" scale and explore and work with transit bounds data at, for example, the county and census tract scale in the United States.
 
```

Comment:
If you want, on my blog 9http://kuanbutts.com/2017/10/12/transitland-boxes/) I did add in one more image, here: https://raw.githubusercontent.com/kuanb/kuanb.github.io/master/images/_posts/transitland-boxes/world.png

Similarly, I also generated this image today, should it be of interest (could be added in the US section):
https://pbs.twimg.com/media/DNAU1e0UEAAvgbI.png:large